build.gradle

```diff
@@ -185,11 +185,9 @@ shadowJar {
     relocate 'com.google', 'shadow.com.google'
     relocate 'com.nimbusds', 'shadow.com.nimbusds'
     relocate 'net.jcip', 'shadow.net.jcip'
-    relocate 'net.minidev', 'shadow.net.minidev'
```

Comment:
This is related to the human flow.  I was testing it manually to verify all is well and I get an error ☹️.  So this must have been broken since the shadow change went in:
```
Caused by: java.lang.NoClassDefFoundError: shadow/net/minidev/json/JSONAware
```